sql/updates/world/master/8888_88_88_88_world.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM `spell_script_names` WHERE `spell_id`=167105 AND `ScriptName`='spell_warr_colossus_smash';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (167105, 'spell_warr_colossus_smash');
```

Comment:
sorry tkrokli i thinked my next commit will not be added here so i added another sql file
^_^